packages/edit-post/src/components/preview-options/index.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	Dropdown,
+	Icon,
+	MenuGroup,
+	MenuItem,
+	Polygon,
+	SVG,
+} from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { PostPreviewButton } from '@wordpress/editor';
+import { __, _x } from '@wordpress/i18n';
+
+const downArrow = (
+	<SVG
+		width="18"
+		height="18"
+		xmlns="http://www.w3.org/2000/svg"
+		viewBox="0 0 18 18"
+	>
+		<Polygon points="9,13.5 14.7,7.9 13.2,6.5 9,10.7 4.8,6.5 3.3,7.9 " />
+	</SVG>
+);
+
+export default function PreviewOptions( {
+	forceIsAutosaveable,
+	forcePreviewLink,
+} ) {
+	const { setPreviewDeviceType } = useDispatch( 'core/block-editor' );
+
+	const deviceType = useSelect( ( select ) => {
+		return select( 'core/block-editor' ).getPreviewDeviceType();
```

Comment:
Could we have a filter to be able to set the default view that authors see when loading the page? Several clients I know of would like to have a mobile-first experience, while other would like desktop view to be the default.